components/bl-credit-card-component/src/card-preview.js

```diff
@@ -0,0 +1,110 @@
+import { useMemo } from 'react';
+
+const InitialCardProps = {
+  expirationLocaleHeader: 'valid thru',
+  namePlaceholder       : 'YOUR NAME HERE',
+  issuer                : 'unknown',
+  numberLength          : 16,
+  numberMaxLength       : 19,
+};
+
+const NumberSpaces = { shortFormat: [4, 11], defaultFormat: [4, 9, 14] };
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export function CardPreview(props) {
+  const { cardNumber, expiry, cvc, focused, name, card, cvcVisibility } = props;
+
+  const { creditCardNumber, expirationDate, issuer } = useCardPreview(cardNumber, expiry, card);
+
+  const currentCVC = useMemo(() => cvcVisibility ? cvc : cvc.replace(/\d/g, '*'), [cvc, cvcVisibility]);
```

Comment:
Will be good if you create constants for some patters.
For example:
```
const RegexPatterns = {
  ALL_DIGITS: '/\d/g',
  ALL_SPACES: '/ /g'
}
```
Or create a function that describe what we do

```const hideDigits = string => string.replace(/\d/g, '*')```

Because I see a lot of work with strings, but not all of them is pretty simple to catch on the fly